result_specification.md

```diff
@@ -0,0 +1,85 @@
+# Result Specification
+
+## Generation
+
+#### Generation Result
+
+The Generation Result is dedicated in the file `Generation.json`. File contains:
+
+- model_id: the reference id for the generative model
+- prompt_template: the template for the prompt for generate the result
+- result:
+
+    - reference_index: paper index given in the dataset
+    - DOI: DOI reference for the paper
+    - generation: generated result
+
+#### Ground Truth
+
+The Ground Truth is dedicated in the file `Ground_Truth.json`. File contains:
+
+- reference_index: paper index given in the dataset
+- DOI: DOI reference for the paper
+- ground_truth: annotated result
+
+#### Generation Range
+The Generation range for each item is given below:
+
+```
+    "Light_source": ["UV", "Solar", "UV-Vis", "Monochromatic", "Solar Simulator"],
+    "Lamp" : ["Fluorescent", "Mercury", "Halogen", "Mercury-Xenon", "LED", "Tungsten", "Xenon", "Tungsten-Halide", "Solar Simulator"],
+    "Reactor_type": ["Slurry", "Fixed-bed", "Optical Fiber", "Monolithic", "Membrane", "Fluidised-bed"],
+    "Reaction_medium": ["Liquid", "Gas"],
+    "Operation_mode" : ["Batch", "Continuous", "Batch/Continuous"]
+```
+
+#### Generation Example
+Result based on LLama-3-70B:
+
+```
+generation:
+{
+    "catalyst": " TiO2",
+    "co_catalyst": " Ag",
+    "light_source": " UV",
+    "lamp": " Hg",
+    "reaction_medium": " Liquid",
+    "reactor_type": " Slurry",
+    "operation_mode": " Batch"
+}
+```
+
+
+## Evaluation
+The Average accuarcy for each item, calculated according to `Evaluation Process` in `README`.
+The evaluation result is dedicated in `Evaluation.json`. File contains:

```

Comment:
modles change in time. Shouldn't you add the date when the model was run and its version?